### ISSUE TITLE
Style.BasicStyles.ColorizedText is not working

### ISSUE THREAD

**Issue by visualczm** [rabanti-github/PicoXLSX#8]
hi,
why Style.BasicStyles.ColorizedText is not working in  my project
Style.BasicStyles.ColorizedBackground OK.

help me~~

![image](https://user-images.githubusercontent.com/22953065/44309340-d45e4e80-a3f7-11e8-8e3c-7b533d329e8a.png)
demo10 ok
![image](https://user-images.githubusercontent.com/22953065/44309368-38811280-a3f8-11e8-9c8a-d6aeb44290a0.png)

**Comment by rabanti-github** [comment]
Hi

It is a bug in the style sub-set. It only ssems to happen if the only style information is a foreground color.
I am working on a patch and will update the library as soon as possible.

<strike>
Can you provide the code segment where you try to assign the color (style) to the text?
From the first picture, I only can determine the colorized background. The line with ColorizedText is commented out.

However, there are two attempts:

**1. Only colorized foreground text**
In this case, you coud simply try this code to assign the style to a cell:
```c#
workbook.WS.Value("test", Style.BasicStyles.ColorizedText("AAFFAA"));  
```
**2. Combined style (e.g. foreground and background)**
In this case case, you could use style chaining like this:
```c#
Style chainedStyle = new Style()   // Create a new (blank) style...
                .Append(Style.BasicStyles.ColorizedText("FF00FF"))   // ... and append part 
                .Append(Style.BasicStyles.ColorizedBackground("AAFFAA"));   // ... and append another part 
workbook.WS.Value("test", chainedStyle );
```
Keep in mind that the first three lines of this code are one single function call (could be on one line)

Can you try this out or provide me the relevant code?
</strike>

Thank you in advance.

**Comment by rabanti-github** [comment]
The bug was fixed with commit 3d88769.
The NuGet package is up to date and should be available in a few minutes (v2.5.1)

Thank you for the report.
Please test the functionality and repoen the issue if somethin is not working as expected.

Greetings.

**Comment by visualczm** [comment]
good job.
